exonum/src/explorer/mod.rs

```diff
@@ -512,12 +512,12 @@ pub struct BlockchainExplorer<'a> {
 }
 
 impl<'a> BlockchainExplorer<'a> {
-    /// Creates a new `BlockchainExplorer` instance.
+    /// Create a new `BlockchainExplorer` instance.
```

Comment:
Isn't it common to use 3rd person verb phrase as a summary fragment: https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment 

The standard Rust docs also follow this principle: https://doc.rust-lang.org/std/vec/struct.Vec.html#method.with_capacity

Reply:
I think that should be a limit for nitpicking PRs. Otherwise every PR may become an endless sequense of "little fixes".

According to the pointed problem, I think it's okay. We have a lot of little inconsistencies in the docs. When we'll have a tech writers team for this, we'll solve it. But now I think it's more important to write new code, than support completely brilliant documentation.

Reply:
I am already confused which variant you prefer more. Both have explanations behind them. Just use the one that is more convenient for you. I will follow your preferences in my corrections

Reply:
There is a difference between _API_ documentation (which follows the linked format), and _implementation_ documentation (which is usually written as commands). We use both consistently in appropriate places, with no 'preferences'.

Reply:
We had a similar situation with EJB documentation where my corrections from infinitive to 3rd person forms were rejected for the reason that infinitive is allowable in such cases...

I absolutely agree with what you say about _implementation_ and _API_ docs. However, there **is** a problem in your following these rules. Anyway, we need a style guide and I will develop one for common use to avoid such situations in the future.

Reply:
The EJB docs used and continue to use the infinitive in implementation docs (which is a different species from method API docs). Would you expand on **the** problem — we don't seem to have any with it?

Regarding this particular case, nowhere I said it is anything more than a suggestion — keep whatever you are happy with, as it does not affect the meaning.